### ISSUE TITLE
"Manual View" still correct

### ISSUE THREAD

**Issue by tronta** [AaronErhardt/AaronErhardt.github.io#57]
Here I find a chapter about `manual-vie`:
https://github.com/AaronErhardt/AaronErhardt.github.io/blob/book/src/widget_macro_reference.md#manual-view

Is this still correct?

**Comment by AaronErhardt** [comment]
Good catch. It's not anymore, as you can see here: https://github.com/AaronErhardt/AaronErhardt.github.io/blob/book/src/0_2_to_0_4.md#widget-macro

I still haven't had time for the book. I'm focusing on the 0.5 release right now, I hope things will settle down a bit soon...

**Comment by tronta** [comment]
Don't worry. Thanks for pointing me there 
If you think, there will be not too many changes, I'll can propose a change.

**Comment by AaronErhardt** [comment]
There shouldn't be a lot of changes. But as always, feel free to contribute only as much as you want. It doesn't need to be complete right away.
